packages/create-webpack-app/templates/init/vue/src/App.vue.tpl

```diff
@@ -0,0 +1,18 @@
+<template>
+  <div id="app">
+    <img alt="Webpack logo" src="./assets/webpack.png">
+    <HelloWorld/>
+  </div>
+</template>
+
+<script<% if (langType === 'Typescript') { %> lang="ts"<% } %>>
+import { defineComponent } from 'vue'
+import HelloWorld from './components/HelloWorld.vue'
+
+export default defineComponent({
```

Comment:
This is good, but lets go for composition api

https://vuejs.org/guide/extras/composition-api-faq.html